routes/query.js

```diff
@@ -0,0 +1,26 @@
+module.exports = (app, pool) => {
```

Comment:
So this is the route for production queries correct?

Reply:
Yes, it pulls the 'production' or 'live site' set of data. query-staging pulls the 'preview' data. The terminology is confusing, but the concept is simple. The 'production' site can use 'preview' DATA, but here 'preview' refers only to the data itself.

Reply:
Awesome thank you for explaining!